README.md

```diff
@@ -1,4 +1,4 @@
-# WebSocket Client v0.3.1
+# WebSocket Client v0.4.0
```

Comment:
The changes look more like a major version increment. I would make the version 1.0.0